libraries/AP_RPM/AP_RPM.h

```diff
@@ -104,6 +104,9 @@ class AP_RPM
 
     static AP_RPM *get_singleton() { return _singleton; }
 
+    static constexpr uint32_t update_interval_hz = 10;
```

Comment:
@magicrub this will need to be 40 hz for heli. I am still working on auto tests that ensure changes don’t break heli. This is one that I will have to independently check to ensure you didn’t break anything. I will add myself as a reviewer so that I can check before it gets merged.

Reply:
good news, I preserved the 40Hz for copter